### ISSUE TITLE
Unable to generate development documentation when testing pr9968 with NVDA source

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#10702]
### Steps to reproduce:
First open cmd
then cd nvda
Last use scons devDocs command generates developer documentation

### Actual behavior:
Generate error
### Expected behavior:
Generate success
#### NVDA installed/portable/running from source:
NVDA source
#### NVDA version:
none
#### Windows version:
windows10-1909
#### Name and version of other software in use when reproducing the issue:
none
#### Other information about your system:
 visual studio 2019
python3.7.6+virtualenv
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
none
error log:
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
interfaceAction(["source\comInterfaces\AcrobatAccessLib.py", "source\comInterfaces\__pycache__\AcrobatAccessLib.cpython-
37.pyc"], ["source\typelibs\AcrobatAccess.tlb"])
scons: *** [source\comInterfaces\AcrobatAccessLib.py] TypeError : item 2 in _argtypes_ passes a union by value, which is
 unsupported.
Traceback (most recent call last):
  File "D:\Program Files\Python\Python37-32\lib\ctypes\__init__.py", line 121, in WINFUNCTYPE
    return _win_functype_cache[(restype, argtypes, flags)]
KeyError: (<class 'ctypes.HRESULT'>, (<class 'ctypes.c_long'>, <class 'comtypes.automation.tagVARIANT'>), 0)
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "E:\Source\Repos\screenReader\nvda\include\scons\src\engine\SCons\Action.py", line 1202, in execute
    result = self.execfunction(target=target, source=rsources, env=env)
  File "E:\Source\Repos\screenReader\nvda\source\comInterfaces_sconscript", line 35, in interfaceAction
    comtypes.client.GetModule(source)
  File "E:\Source\Repos\screenReader\nvda\include\comtypes\comtypes\client\_generate.py", line 110, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "E:\Source\Repos\screenReader\nvda\include\comtypes\comtypes\client\_generate.py", line 184, in _CreateWrapper
    mod = _my_import(fullname)
  File "E:\Source\Repos\screenReader\nvda\source\comInterfaces_sconscript", line 26, in new_my_import
    return old_my_import(fullname)
  File "E:\Source\Repos\screenReader\nvda\include\comtypes\comtypes\client\_generate.py", line 24, in _my_import
    return __import__(fullname, globals(), locals(), ['DUMMY'])
  File "E:\Source\Repos\screenReader\nvda\source\comInterfaces\_C523F390_9C83_11D3_9094_00104BD0D535_0_3_0.py", line 303
, in <module>
    ( ['in'], VARIANT, 'value' )),
  File "E:\Source\Repos\screenReader\nvda\include\comtypes\comtypes\__init__.py", line 329, in __setattr__
    self._make_methods(value)
  File "E:\Source\Repos\screenReader\nvda\include\comtypes\comtypes\__init__.py", line 698, in _make_methods
    prototype = WINFUNCTYPE(restype, *argtypes)
  File "D:\Program Files\Python\Python37-32\lib\ctypes\__init__.py", line 123, in WINFUNCTYPE
    class WinFunctionType(_CFuncPtr):
TypeError: item 2 in _argtypes_ passes a union by value, which is unsupported.
scons: building terminated because of errors.
thanks

**Comment by Adriani90** [comment]
cc: @feerrenrut

**Comment by CyrilleB79** [comment]
@dingpengyu what is your version of Python? Anything to do with #10696? It's just an idea, since I did not try to build the documentation by myself.

**Comment by feerrenrut** [comment]
Please see pinned issue #10696. There is currently an issue building with Python 3.7.6, please use 3.7.5 instead.
